python/__init__.py.in

```diff
@@ -37,6 +37,17 @@ else:
 if openravepy_currentversion is not None:
     from sys import modules
     modules['openravepy'] = openravepy_currentversion
+    # caused modules reloaded; has to reimport sys.modules
+    del modules
```

Comment:
@cielavenir what's the error when you remove `del modules` but keep everything else?

Reply:
@cielavenir ping

Reply:
short: that is not required

long:

- after `modules['openravepy'] = openravepy_currentversion`, `modules` becomes None only on Py2.
- (in `__init__.py`) we read `modules['openravepy'].__dict__` only on Py3, modules becoming None can be ignored **miraculously**.

Reply:
thank you for checking